display_information/display-dimensions/src/main/java/com/esri/samples/display_dimensions/DisplayDimensionsSample.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.display_dimensions;
+
+import com.esri.arcgisruntime.layers.DimensionLayer;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.MobileMapPackage;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.util.Objects;
+
+public class DisplayDimensionsSample extends Application {
+
+  private DimensionLayer dimensionLayer;
+  private MapView mapView;
+  private MobileMapPackage mobileMapPackage;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(Objects.requireNonNull(getClass().getResource("/display_dimensions/style.css")).toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Dimensions Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // set up display label for dimension layer name, and check boxes for controlling visibility and definition expression
+      Label dimensionLayerName = new Label();
+      CheckBox visibilityCheckBox = new CheckBox("Dimension Layer visibility");
+      visibilityCheckBox.setSelected(true);
+      CheckBox defExpressionCheckBox = new CheckBox("Definition Expression:" + "\n" + "Dimensions >= 450m");
```

Comment:
Definition expression text has ellipsis on Windows "Definition Expressio....". Widen the UI box (won't block much of the map)
![image](https://user-images.githubusercontent.com/56556822/145586184-258b87f3-fc0c-4e2e-829f-b75885d710df.png)

Reply:
i think this is the height property, I've tweaked it: can you let me know if you see as follows?
<img width="256" alt="Screenshot 2021-12-13 at 12 00 56" src="https://user-images.githubusercontent.com/36415565/145808664-c264a5d2-16a8-48c3-9d85-f04909ca0206.png">

Reply:
Looks good 👍